apps/examples/testcase/ta_tc/media/itc/itc_media_fileinputdatasource.cpp

```diff
@@ -141,8 +149,8 @@ static void itc_media_FileInputDataSource_open_n(void)
 {
 	media::stream::FileInputDataSource source(file_path);
 	TC_ASSERT_EQ("open", source.open(), 1);
-	for(int i=0; i<COUNT; i++){
-		TC_ASSERT_EQ("open", source.open(), 0);
+	for (int i=0; i<COUNT; i++) {
```

Comment:
Please keep Space between operator and operands, there are a lot of cases where its not done. 
Kindly check all of them.

Reply:
changes done and code also updated in gerrit